actionview/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 ## Rails 4.2.11.1 (March 11, 2019) ##
 
-*   No changes.
+*   Only accept formats from registered mime types
+
+    A lack of filtering on mime types could allow an a attacker to read
```

Comment:
I think there is a typo. It should be `an attacker`.

Reply:
Ditto. This was fixed in 5-2-stable.

@jhawthorn would you mind updating this to fix the typo?